test/client/fixtures/sandbox/node/classes-test.js

```diff
@@ -22,6 +23,160 @@ if (window.PerformanceNavigationTiming) {
     });
 }
 
+test('window.Blob([data], { type: \'\' }) should return correct result for all possible data type cases (ArrayBuffer, TypedArray, ...) (GH-1599)', function () {
+    var testCases = [
+        // Image types
+        {
+            mime:      'image/x-icon',
```

Comment:
You should not validate mime types. You need to check all Array like types as blob arguments.
Rewrite test as:
```js
Pseudo code

var data = {
    mime: '...',
    signature: '...'
};

['ArrayBuffer', 'Uint8Array' and etc.].forEach(type => {
    data = ... //convert data to an appropriate type
    check(window.Blob(data) 
} )
```